ghost/i18n/locales/de/comments.json

```
@@ -61,7 +61,7 @@
     "Show 1 previous comment": "1 vorherigen Kommentar anzeigen",
     "Show comment": "Kommentar anzeigen",
     "Sign in": "Anmelden",
```

Found this variation in `portal.json` -- adjusting for consistency. 

```suggestion
    "Sign in": "Einloggen",
```